deepchem/package-lock.json

```diff
@@ -0,0 +1,4761 @@
+{
```

Comment:
Sir, as per the sources we looked up online, it is highly suggested to include the package-lock.json file in the repository so that anyone who runs `npm i` can get the exact same dependency tree that we were developing on. It is an auto generated file.